packages/ibm-products/src/components/GetStarted/GetStarted.mdx

```diff
@@ -0,0 +1,52 @@
+import { Story, ArgsTable, Canvas } from '@storybook/addon-docs';
+import { CodesandboxLink } from '../../global/js/utils/story-helper';
+import { GetStarted } from '.';
+import * as stories from './GetStarted.stories';
+
+# GetStarted
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Example usage](#example-usage)
+- [Component API](#component-api)
+
+## Overview
+
+<Canvas>
+  <Story of={stories.Default} />
+</Canvas>
+
+## Example usage
+
+### with Pictogram
+
+<Canvas>
+  <Story of={stories.withPictogram} />
+</Canvas>
```

Comment:
Not seeing a difference in default and pictogram so wonder if we need to have these listed separately?